packages/playground/app/assets/js/webcomponentsjs/package.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
package.json as well.

Scrolling through the files I am wondering is there a better way to consume the library instead of copy the files directly to the project